lms/envs/devstack_with_worker.py

```diff
@@ -19,6 +21,29 @@
 # Require a separate celery worker
 CELERY_ALWAYS_EAGER = False
 
+# Periodic tasks can be created automatically at project start (when celery process is started).
+# To not create them manually in admin (but they still will be shown in admin),
+# we can define them in settings.py in such way.
+CELERYBEAT_SCHEDULE = {
+    'student_counter': {
+    'task': 'openedx.core.djangoapps.global_statistics.tasks.count_data',
+    'schedule': 20,
+    }
+}
+
+# URL to send data within periodic task processing.
+PERIODIC_TASK_POST_URL = 'http://192.168.1.139:7000/receive/'
+
+# Geographical coordinates of the eDX platform server location in decimal degrees up to the city.
+# Please use these settings if your eDX server's location differ from its physical location.
+# For example, you use cloud services like Amazon(AWS), Microsoft(Azure), etc.
+# Otherwise, leave these settings as is.
+
+# Example of usage: 'Europe/Kiev' PLATFORM_LATITUDE = '50.4546600', PLATFORM_LONGITUDE = '30.5238000'
+
+PLATFORM_LATITUDE = ''
+PLATFORM_LONGITUDE = ''
```

Comment:
I think it will be more convenient to use same as dango time, like: region/city: # Example of usage: 'Europe/Kiev'